src/components/tearComponent/tears.jsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react'
```

Comment:
Perhaps split these up into separate files, ie one rip per file? Pop them in a sub-folder here should be fine.